praw/objector.py

```diff
@@ -134,6 +134,16 @@ def _objectify_dict(self, data):
         elif {"mod_permissions", "name", "sr", "subscribers"}.issubset(data):
             data["display_name"] = data["sr"]
             parser = self.parsers[self._reddit.config.kinds["subreddit"]]
+        elif {"mod_action_data", "user_note_data"}.issubset(data):
+            data["user"] = self._reddit.redditor(data["user"])
+            data["moderator"] = self._reddit.redditor(data["operator"])
+            data["subreddit"] = self._reddit.subreddit(data["subreddit"])
```

Comment:
```suggestion
            data["moderator"] = self._reddit.redditor(data["operator"])
            data["subreddit"] = self._reddit.subreddit(data["subreddit"])
            data["user"] = self._reddit.redditor(data["user"])
```
Extreme nit: sort these